src/scripts.js

```diff
@@ -60,30 +61,39 @@ function updateBookingButtons(bookingBtns) {
   });
 }
 
+logOutBtn.addEventListener('click', logout);
+goHomeBtn.addEventListener('click', showHomepage);
+
+
+//~~~~~~~~~~~~~~~~~~ functions ~~~~~~~~~~~~~~~~~~~
+function fetchData(logNameCheck2) {
+  const response = Promise.all([fetchSingleUser(logNameCheck2), fetchRoomsData(), fetchBookingsData()])
+  return response;
+}
+
+// this can be used for the manager login
+function fetchAllData() {
+  const response = Promise.all([fetchUsersData(), fetchRoomsData(), fetchBookingsData()])
+  return response;
+}
+
 function logIn(e) {
     e.preventDefault();
     let logNameCheck = loginName.value
     let logPasswordCheck = loginPassword.value
-    console.log('1st grab of name', logNameCheck)
-    console.log('1st grab of password', logPasswordCheck)
     let logNameCheck2 = parseInt(logNameCheck.substring(8))
-    loginName.value = ''
-    loginPassword.value = ''
-    customerLookUp(logNameCheck2, logPasswordCheck);
+    clearLoginValues()
+    customerLookUp(logNameCheck2, logPasswordCheck, logNameCheck);
 
 }
 
-function customerLookUp(logNameCheck2, logPasswordCheck) {
-    if(logNameCheck2 > 0 && logNameCheck2 <= 50 && logPasswordCheck === 'overlook2021') {
+function customerLookUp(logNameCheck2, logPasswordCheck, logNameCheck) {
+    if(logNameCheck2 > 0 && logNameCheck2 <= 50 && logNameCheck.startsWith('customer') && logPasswordCheck === 'overlook2021') {
```

Comment:
This conditional could also be broken down into some helper functions that you can call in the if statement if you want to shorten it up at all.